### ISSUE TITLE
APIClassNotFound Exception

### ISSUE THREAD

**Issue by mathmatrix828** [paquettg/leaguewrap#117]
Using CodeIgniter, I get a Class Not Found exception when trying to use the ChampionMastery class.
I am using CI to import from composer

An uncaught Exception was encountered

Type: LeagueWrap\Exception\ApiClassNotFoundException

Message: The api class "LeagueWrap\Api\Championmastery" was not found.

Filename: /var/www/lol/application/vendor/paquettg/leaguewrap/src/LeagueWrap/Api.php

Line Number: 136

Backtrace:

File: /var/www/lol/application/controllers/Summoner.php
Line: 29
Function: __call

File: /var/www/lol/application/controllers/Summoner.php
Line: 29
Function: Championmastery

File: /var/www/lol/index.php
Line: 315
Function: require_once

Coming from
```php
use LeagueWrap\Api;

class Summoner extends CI_Controller {
  function __construct() {
    parent::__construct();
    $this->api = new API('my-key');
  }

  function index() {
    $api = $this->api;
    ...
    $master = $api->championMastery();
    ...
    (Gets rendered in view)
  }
}
```

**Comment by mathmatrix828** [comment]
Issue appears to occur when using matchList as well

**Comment by mathmatrix828** [comment]
Looks like it may be because the files don't exist!

**Comment by mathmatrix828** [comment]
Issue appears to be that package version available on Composer is 0.6.2
